prometheus_api_client/prometheus_connect.py

```diff
@@ -60,6 +62,7 @@ def __init__(
         self.url = url
         self.prometheus_host = urlparse(self.url).netloc
         self._all_metrics = None
+        self._timeout = timeout
```

Comment:
@FRosner instead of changing it in every query function, I believe it might easier to update the session options when initializing the session object here: https://github.com/4n4nd/prometheus-api-client-python/blob/c77c0b91d85a709848e72b457ef26b706a1ceccb/prometheus_api_client/prometheus_connect.py#L73-L81

Reply:
I was under the impression that you could pass a timeout parameter when initializing a session object. But it seems like there isn't a way to do it. your suggested way seems to be the easiest one 👍